tortoisebeacon/handlers.go

```diff
@@ -319,7 +320,7 @@ func (tb *TortoiseBeacon) verifyFirstVotingMessage(message FirstVotingMessage, c
 	defer tb.consensusMu.Unlock()
 
 	if tb.hasVoted[firstRound] == nil {
-		tb.hasVoted[firstRound] = make(map[nodeID]struct{})
+		tb.hasVoted[firstRound] = make(map[nodePK]struct{})
```

Comment:
nodePK alias to string doesn't make much sense to me too

Reply:
Removed